.changeset/two-walls-cheer.md

```
@@ -0,0 +1,5 @@
+---
+"@vahor/typed-es": patch
+---
+
+Add `ip_range` bucket agg support.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)